source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
+Relational Migrator displays diagram names on the ribbon tab. 
+
+About this Task
+---------------
+
+You can rename diagrams to customize your diagram properties. Relational
+Migrator shows your diagram names on the ribbon tab of the Mapping view and in
+the Diagram Properties pane. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename. 
```

Comment:
[nit] According to the [style guide](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/task/#steps) (and [previous team discussion](https://docs.google.com/document/d/14dbVyMTsn42pGpIl41aFWC4T7IFSvGBFdWjmsXPJw6w/edit)), we don't punctuate step titles.
```suggestion
   .. step:: Navigate to the diagram you want to rename
```

Reply:
Done!

Reply:
Done!